backup/moodle2/backup_hvp_stepslib.php

```diff
@@ -15,68 +15,107 @@
 // along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
 
 /**
- * @package moodlecore
- * @subpackage backup-moodle2
- * @copyright 2010 onwards Eloy Lafuente (stronk7) {@link http://stronk7.com}
+ * @package mod_hvp
+ * @copyright 2016 Mediamaisteri Oy {@link http://www.mediamaisteri.com}
```

Comment:
I'm not an OS lawyer, so I'm not completely sure who should be mentioned on this line. :P Perhaps there should be at least a mention of the original author? Or perhaps this isn't really a problem in the first place.